app/models/patient.rb

```diff
@@ -977,11 +991,9 @@ def self.calc_current_age_base(provided_date_of_birth: nil)
   end
 
   # Determine the proper language for sending reports to this monitoree
-  def select_language
+  def self.select_language
```

Comment:
Remove the `self.` -- this is used many times as a patient instance method in the PatientMailer.

Reply:
fixed!